dev/PipsPager/PipsPager.cpp

```diff
@@ -521,6 +528,49 @@ void PipsPager::OnNextButtonClicked(const IInspectable& sender, const winrt::Rou
     SelectedPageIndex(SelectedPageIndex() + 1);
 }
 
+void PipsPager::OnPipsAreaGettingFocus(const IInspectable& sender, const winrt::GettingFocusEventArgs& args)
+{
+    if (const auto repeater = m_pipsPagerRepeater.get())
+    {
+        // Easiest way to check if focus change came from within:
+        // Check if element is child of repeater by getting index and checking for -1
+        // If it is -1, focus came from outside and we want to get to selected element.
+        if (const auto oldFocusedElement = args.OldFocusedElement().try_as<winrt::UIElement>())
+        {
+            if (repeater.GetElementIndex(oldFocusedElement) == -1)
+            {
+                if (const auto realizedElement = repeater.GetOrCreateElement(SelectedPageIndex()).try_as<winrt::UIElement>())
```

Comment:
Is it possible SelectedPageIndex() returns -1

Reply:
no, it is not possible:)